life_expectancy/tests/conftest.py

```diff
@@ -2,12 +2,12 @@
 import pandas as pd
 import pytest
 
+from life_expectancy.cleaning import load_data
```

Comment:
Don't use functions from your packages in the test setup module. If you eventually change it, your tests may suffer unintended consequences (e.g. they may pass when they should fail). As a rule of thumb, the test suit should not depend on fixtures that depend on the things it's trying to test.

Reply:
Got it, changed it in the code.